app/controllers/transactions_controller.rb

```diff
@@ -81,6 +83,8 @@ def transaction_params
 
   def set_transaction
     @transaction = Transaction.find(params[:id])
+  rescue ActiveRecord::RecordNotFound
+    redirect_to transactions_path, notice: "Draft transaction was successfully deleted"
```

Comment:
Handle this by showing a 404 page.  The notice message is not always correct.  The user can use any number on the url and it will go here.  For example: https://localhost:3000/transactions/892374923784

Reply:
Done